mobile-app/App.tsx

```diff
@@ -4,16 +4,44 @@ import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import { LocalScreen } from './src/screens/LocalScreen';
 import { HomeScreen } from './src/screens/HomeScreen';
 import type { RootStackParamList } from './src/screens/navigationTypes';
+import { AuthState } from './src/services/authService';
+import Login from './src/components/Login/Login';
+import { deleteJwt, getJwt, saveJwt } from './src/services/deviceStorageService';
+import { RemoteScreen } from './src/screens/RemoteScreen';
 
 const RootStack = createNativeStackNavigator<RootStackParamList>();
 
 export default function App() {
+  const [authState, setAuthState] = React.useState<AuthState>({ jwt: null });
+
+  React.useEffect(() => {
+    getJwt().then((jwt) => {
+      console.log(`App - Got JWT Token: ${jwt}`);
+      setAuthState({ jwt });
+    });
```

Comment:
You have no time to improve this behavior but generally we prefer using React context provider